src/types/viewport.cpp

```diff
@@ -6,6 +6,11 @@
 
 using namespace Microsoft::Console::Types;
 
+Viewport::Viewport() noexcept :

```

Comment:
I didn't intend to add this. But the compiler complains about there is no default ctor when initializing `til::some<Viewport, 4>result`.